yolox/cli/export_onnx.py

```diff
@@ -1,21 +1,28 @@
 # Copyright (c) Megvii, Inc. and its affiliates.
 
 import argparse
-import os
-from loguru import logger
+import logging
+from pathlib import Path
 
 import torch
+from loguru import logger
 from torch import nn
 
-from yolox.exp import get_exp
+from yolox.models import Yolox
 from yolox.models.network_blocks import SiLU
 from yolox.utils import replace_module
+import onnx
+from onnxsim import simplify
+
+logger = logging.getLogger(__name__)
 
 
 def make_parser():
     parser = argparse.ArgumentParser("yolox onnx-deploy")
+    parser.add_argument("-n", "--name", type=str, default=None, help="model name")
+    parser.add_argument("-c", "--ckpt", type=str, default=None, help=".pth file path")
     parser.add_argument(
-        "--output-name", type=str, default="yolox.onnx", help="output name of models"
+        "--onnx_name", type=str, default="yolox.onnx", help="name of onnx model"
```

Comment:
`--onnx-name` so we're consistent in using hyphens as the CLI argument naming convention